theano/sandbox/cuda/opt.py

```diff
@@ -1128,11 +1128,9 @@ def local_gpu_advanced_incsubtensor1(node):
             compute_capability = device_properties(active_device_no)['major']
             if (compute_capability < 2 or y.ndim != 2 or x.ndim != 2):
 
-                gpu_op = GpuAdvancedIncSubtensor1(
-                    set_instead_of_inc=set_instead_of_inc)
+                gpu_op = tensor.AdvancedIncSubtensor1(**node.op._props_dict())
             else:
-                gpu_op = GpuAdvancedIncSubtensor1_dev20(
-                    set_instead_of_inc=set_instead_of_inc)
+                gpu_op = GPUAdvancedIncSubtensor1_dev20(**node.op._props_dict())
```

Comment:
I don't know why, but PEP8 complained if i just used `GPUAdvancedIncSubtensor1_dev20`. That's why i did the import as `theano.sandbox.cuda.basic_ops.GPUAdvancedIncSubtensor1_dev20`